src/lib/forms/rjsf/RJSForm.js

```diff
@@ -56,6 +56,18 @@ export class RJSForm extends Component {
     };
   }
 
+  componentDidMount() {
+    const inputs = document.getElementsByTagName('input');
+
+    for (let input of inputs) {
+      const isRJSFInput = input.id.includes('root');
+      if (isRJSFInput) {
+        input.onkeydown = (event) =>
+          event.key === 'Enter' ? event.preventDefault() : null;
```

Comment:
Does returning `null` fallback to the default behavior?

Reply:
good one! From what i understand, it does, because the original event is not overridden/prevented (as is the case with preventDefault())

Reply:
sure! 

Yes i did, everything works as normal :smiley: